app/controllers/atmosphere/admin/virtual_machine_templates_controller.rb

```diff
@@ -17,13 +21,26 @@ def show
   def edit
   end
 
+  # GET /virtual_machine_templates/1/share
+  def share
+  end
+
+  # POST /virtual_machine_templates/1/export
+  def export
```

Comment:
You proposed to find @Nuanda solution somewhere [here](https://github.com/dice-cyfronet/atmosphere/tree/master/app/controllers/atmosphere/admin)? I cannot find them in [history](https://github.com/dice-cyfronet/atmosphere/commits/master/app/controllers/atmosphere/admin).